packages/core/src/config/load.ts

```diff
@@ -111,18 +111,19 @@ export async function getConfig(configPath: string | undefined = findConfig()):
 
 interface CreateConfigOptions {
   extends?: string[];
-  tokens?: RegionalTokenWithValidity[],
+  tokens?: RegionalTokenWithValidity[];
 }
 
-export async function createConfig (content: string, options?: CreateConfigOptions): Promise<Config>;
-export async function createConfig (rawConfig: RawConfig, options?: CreateConfigOptions): Promise<Config>;
-export async function createConfig (rawConfig: any, options: CreateConfigOptions = {}): Promise<Config> {
-  if (typeof rawConfig === 'string') {
-    rawConfig = parseYaml(rawConfig);
+export async function createConfig(
+  config: string | RawConfig,
+  options?: CreateConfigOptions
+): Promise<Config> {
+  if (typeof config === 'string') {
+    config = parseYaml(config) as RawConfig;
```

Comment:
Can you avoid mutating the `config` argument?

Reply:
Changed